app/src/androidTest/java/org/mozilla/fenix/glean/BaselinePingTest.kt

```diff
@@ -87,9 +98,15 @@ class BaselinePingTest {
         // Move it to background.
         device.pressHome()
 
+        // Due to bug 1632184, we need move the activity to foreground and then to background
```

Comment:
As I read this this only puts it in foreground again.
So this sentence is slightly wrong or at least not matching the code below.